### PR TITLE
Adding state_snapshot grpc stubs to Python Clientservice. These were missing from the Python ClientService class.

### PR DESCRIPTION
**Pull request by alfredd**
* **Problem Overview**  
state_snapshot grpc bindings were missing from the ClientService python class under Apollo tests. 

The changes include 
1. Adding a line to the update_python_bindings.py file that allows the grpc bindings for state_snapshot.proto to be generated. 
2. Generating and adding the grpc python bindings for changes to the .proto files. 
3. Adding code for the state_snapshot generated bindings in the csutil.py file.

### CODE REVIEW

File: tests/apollo/clientservice/csutil.py [hunk 1]
```diff
@@ -51,3 +54,16 @@ def subscribe(self, id, legacy=False):
...
+        return self.state_snapshot_stub.GetRecentSnapshot(req)
+
+
+    def stream_snapshot(self, id, last_recvd_key=""):
+        req = state_snapshot_proto.StreamSnapshotRequest(snapshot_id=id, last_received_key=last_recvd_key)
+        return self.state_snapshot_stub.StreamSnapshot(req)
+
+    def read_as_of(self, id, keys_list=[""]):
+        req = state_snapshot_proto.ReadAsOfRequest(snashot_id = id, keys = keys_list)
+        return self.state_snapshot_stub.ReadAsOf(req)
```
Comment (shgandhi) [inline tests/apollo/clientservice/csutil.py:69]: Please add newline at end of file.

**Review by teoparvanov — APPROVED** [review]
LGTM, thanks @alfredd ! You can merge this after addressing Shruti's comment about the commit messages.

**Comment by alfredd** [comment]
> Thanks for the change @alfredd. LGTM. However, please change your commit messages according to the guidelines mentioned here - https://cbea.ms/git-commit/.

@shgandhi I've added a new line and updated my comment.

**Comment by alfredd** [comment]
I'm closing this pull request as it has become unnecessarily convoluted. I will create another pull request with proper comments.

**Pull request closed without merging**